src/main/java/com/synopsys/integration/alert/component/authentication/actions/AuthenticationTestAction.java

```diff
@@ -67,9 +67,9 @@ public MessageResult testConfig(String configId, FieldModel fieldModel, FieldAcc
         boolean samlEnabled = registeredFieldValues.getBooleanOrFalse(AuthenticationDescriptor.KEY_SAML_ENABLED);
         if (!ldapEnabled && !samlEnabled) {
             String errorMessage = "Enable LDAP or SAML authentication.";
-            Map<String, AlertFieldStatus> errorsMap = Map.of(
-                AuthenticationDescriptor.KEY_LDAP_ENABLED, AlertFieldStatus.error(errorMessage),
-                AuthenticationDescriptor.KEY_SAML_ENABLED, AlertFieldStatus.error(errorMessage));
+            List<AlertFieldStatus> errorsMap = List.of(
```

Comment:
Let's rename this to just `errors` because it is no longer a Map. It also helps if we change the type in the future so that it's not coupled to a specific data structure.